### ISSUE TITLE
IsReachable methods return false on iOS (Local Machine)

### ISSUE THREAD

**Issue by espilioto** [jamesmontemagno/ConnectivityPlugin#91]
This test snippet works fine on Android but fails on iOS:

```
if (await CrossConnectivity.Current.IsRemoteReachable("http://xxx.yyy.zz".Remove(0, 7)))
{
     return true;
}
else if (await CrossConnectivity.Current.IsReachable("http://xxx.yyy.zz".Remove(0, 7)))
{
     return true;
}
else
{
     return false;
}
```

I use both methods because I must test in and out of my local network.
The server is of course reachable from the simulator's browser.
Am I missing something?

**Comment by jamesmontemagno** [comment]
What are you trying to reach? is it a local server?

**Comment by espilioto** [comment]
Yup, a local database.

**Comment by jamesmontemagno** [comment]
what address are you passing in?

**Comment by espilioto** [comment]
Tried both the domain and ip, same result.
Works on android and fails on iOS.

**Comment by jamesmontemagno** [comment]
The code does not currently support it from what I am looking at. Will get back to you.

**Comment by espilioto** [comment]
Roger.
Thanks for your awesome work btw, xamarin.forms wouldn't be on my radar without your plugins :D

**Comment by jamesmontemagno** [comment]
Fixed in 4.0.0-beta and now uses Apple's simple ping for iOS/Mac
